installer/Windows Server 2019/installer/starfish.cmd

```diff
@@ -0,0 +1,3 @@
+@echo off
+cd "C:\Users\Public\Starfish"
```

Comment:
haven't heard about C:\Users\Public before - is that intended to be used for this kind of installation ?

Reply:
That's a public location accessible to all users.
Instead of `C:\Users\Public` we can use %USERPROFILE% which leads to home directory for the current user.
If I echo on my console `echo %USERPROFILE% ` we'll get `C:\Users\Fahad Israr`